package.json

```diff
@@ -8,6 +8,7 @@
     "react-dom": ">=0.14.0 || ^0.14.0-beta3"
   },
   "devDependencies": {
```

Comment:
I'm not 100% sure. Doesn't NPM install these when someone uses these libs? Seems unnecessary if thats the case